frontend/src/test/scala/bloop/bsp/BspMetalsClientSpec.scala

```diff
@@ -382,8 +382,11 @@ class BspMetalsClientSpec(
     val scalacOptions = state.scalaOptions(project)._2.items.flatMap(_.options).map { opt =>
       if (!opt.startsWith("-Xplugin:")) opt
       else {
-        opt.split(":") match {
-          case Array(key, value) => s"$key:${value.split(java.io.File.separator).last}"
+        val idx = opt.indexOf(":")
+        opt.splitAt(idx) match {
+          case (key, value) =>
+            val pluginPath = Paths.get(value.tail).getFileName().toString
```

Comment:
`toString` will be covered by the string interpolation
```suggestion
            val pluginPath = Paths.get(value.tail).getFileName()
```